Postman/SurveyQuotas.postman_collection.json

```diff
@@ -0,0 +1,118 @@
+{
+	"info": {
+		"_postman_id": "b5aa0cd7-e932-4317-bbae-9c7ccf972cff",
+		"name": "SurveyQuotas",
```

Comment:
The file name should be `SurveyQuota `or `SurveyQuotaTargets` or part of the `Surveys` collection (as the code has been added to the that service)